src/azul/indexer/document.py

```diff
@@ -277,6 +277,13 @@ def from_index(self, value: T) -> T:
 pass_thru_json: PassThrough[JSON] = PassThrough(es_type=None)
 
 
+class Range(PassThrough[JSON]):
+    pass
```

Comment:
There is more validation we can add here. That the keys are `gte` and `lte`, that there are no other keys.